plugins/sentinelone/komand_sentinelone/actions/update_analyst_verdict/action.py

```diff
@@ -0,0 +1,31 @@
+import insightconnect_plugin_runtime
+from insightconnect_plugin_runtime.exceptions import PluginException
+
+from .schema import UpdateAnalystVerdictInput, UpdateAnalystVerdictOutput, Input, Output, Component
+
+
+class UpdateAnalystVerdict(insightconnect_plugin_runtime.Action):
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name="update_analyst_verdict",
+            description=Component.DESCRIPTION,
+            input=UpdateAnalystVerdictInput(),
+            output=UpdateAnalystVerdictOutput(),
+        )
+
+    def run(self, params={}):
+        incident_type = params.get(Input.TYPE)
+        analyst_verdict = params.get(Input.ANALYST_VERDICT).replace(" ", "_")
+        incidents = self.connection.validate_incidents_exist(list(set(params.get(Input.INCIDENT_IDS))), incident_type)
+        incidents = self.connection.validate_incident_state(incidents, incident_type, analyst_verdict, "analystVerdict")
```

Comment:
Can you add some comments describing what these calls are doing?

Reply:
I'll add a docstring for both functions in `connection.py`.